Lesson_6/Index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="icon" type="image/x-icon" href="CV-logo.png">
+    <title>Lesson 6- Resume</title>
+</head>
+
+<body>
+    <section>
+        <br>
+        <br>
+        <br>
+        <h1 style="font-size: 30px;">Taras Papst</h1>
+        <a href="Hire_me.html">
+            <img src="hireme.png" alt="Hire-Me" style="width: 100px; height: 100px;" />
+        </a>
+        <br>
+        <img src="Taras.jpg " alt="photo " />
+        <h2 style="font-size: 20px; ">Project/Product Concept Expert</h2>
+    </section>
+    <br>
+
+    <section>
+        <h3><u>Work Summary</u></h3>
+        <table border="2 ">
+            <thead>
+                <tr>
+                    <th>Period</th>
+                    <th>Position</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>
+                        <p>2022 - present</p>
```

Comment:
why you need paragraph inside of `td`?

Reply:
that's incorrect to put block element inside of inline element

Reply:
Reworked